endpointmanager/pkg/nppesquerier/populatenpiorgs.go

```diff
@@ -390,40 +391,53 @@ func buildNPIOrgFromNPICsvLine(data NPICsvLine) (*endpointmanager.NPIOrganizatio
 
 // ReadCsv accepts a file and returns its content as a multi-dimentional type
 // with lines and each column. Only parses to string type.
-func readCsv(ctx context.Context, filename string) ([][]string, error) {
+func readCsv(ctx context.Context, filename string) (*csv.Reader, *os.File, error) {
```

Comment:
Suggest renaming function to csvReader or something like that and fixing the function comments

Reply:
Changed the function name and commentation